deploy/charts/dashboards/zfsLocalPV/zfslocalpv.json

```diff
@@ -0,0 +1,1258 @@
+                     "format": "time_series",
+                     "hide": false,
+                     "interval": "",
+                     "intervalFactor": 2,
+                     "legendFormat": "size",
+                     "refId": "B",
+                     "step": 2,
+                     "target": ""
+                  }
+               ],
+               "thresholds": [ ],
+               "timeFrom": null,
+               "timeRegions": [ ],
+               "timeShift": null,
+               "title": "ARC L2 - Size",
+               "tooltip": {
+                  "shared": true,
+                  "sort": 2,
+                  "value_type": "individual"
+               },
+               "type": "graph",
+               "xaxis": {
+                  "buckets": null,
+                  "mode": "time",
+                  "name": null,
+                  "show": true,
+                  "values": [ ]
+               },
+               "yaxes": [
+                  {
+                     "format": "bytes",
+                     "label": "",
+                     "logBase": 1,
+                     "max": null,
+                     "min": null,
+                     "show": true
+                  },
+                  {
+                     "format": "short",
+                     "label": null,
+                     "logBase": 1,
+                     "max": null,
+                     "min": null,
+                     "show": true
+                  }
+               ],
+               "yaxis": {
+                  "align": false,
+                  "alignLevel": null
+               }
+            }
+         ],
+         "title": "ARC L2",
+         "type": "row"
+      }
+   ],
+   "refresh": "10s",
+   "schemaVersion": 27,
+   "style": "dark",
+   "tags": [
+      "OpenEBS",
+      "ZFS-LocalPV"
+   ],
+   "templating": {
+      "list": [
+         {
+            "current": { },
+            "description": null,
+            "error": null,
+            "hide": 0,
+            "includeAll": false,
+            "label": null,
+            "multi": false,
+            "name": "datasource",
+            "options": [ ],
+            "query": "prometheus",
+            "queryValue": "",
+            "refresh": 1,
+            "regex": "",
+            "skipUrlSync": false,
+            "type": "datasource"
+         },
+         {
+            "allValue": null,
+            "current": { },
+            "datasource": "$datasource",
+            "definition": "label_values(node_exporter_build_info{job=~\"node-exporter\"}, instance)",
+            "description": null,
+            "error": null,
+            "hide": 0,
+            "includeAll": false,
+            "label": null,
+            "multi": false,
+            "name": "node",
+            "options": [ ],
+            "query": {
+               "query": "label_values(node_exporter_build_info{job=~\"node-exporter\"}, instance)",
```

Comment:
Will this nested query not cause issue with the old grafana versions like we faced for lvm localpv?